.github/workflows/translations-sync-dw.yml

```diff
@@ -0,0 +1,35 @@
+name: Sync translations in desktop wallet
+
+on:
+  workflow_dispatch:
+  push:
+    paths: ['apps/desktop-wallet/locales/**']
+    branches: [next]
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  sync-translations:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Sync sources and translations
+        uses: crowdin/github-action@v2
+        with:
+          config: 'apps/desktop-wallet/crowdin.yml'
+          upload_sources: true
+          upload_translations: true
```

Comment:
This will upload any translations that were added in the corresponding language JSON files.